api/src/lib.rs

```diff
@@ -1081,19 +1081,13 @@ fn build_headers(config: &Config) -> Result<HeaderMap> {
     Ok(headers)
 }
 
-fn comment_uids_comma_separated_list<'a>(
-    mut comment_uids: impl Iterator<Item = &'a CommentUid>,
-) -> String {
-    // Build `query_uids == ",".join(comment_uids)`
-    let mut query_uids = String::new();
-    if let Some(first_uid) = comment_uids.next() {
-        query_uids.push_str(&first_uid.0);
-        for comment_uid in comment_uids {
-            query_uids.push_str(",");
-            query_uids.push_str(&comment_uid.0);
-        }
-    }
-    query_uids
+fn comment_uid_list_query<'a>(
+    comment_uids: impl Iterator<Item = &'a CommentUid>,
+) -> Vec<(&'static str, &'a str)> {
+    // Return a list of pairs ("id", "a"), ("id", "b"), ...
+    // The http client will turn this into a query string of
+    // the form "id=a&id=b&..."
+    comment_uids.map(|uid| ("id", uid.0.as_str())).collect()
```

Comment:
A small question re: the API format -- there was a separate PR in platform so maybe it was discussed there, but is there a way to avoid repeating `id`? or no because comma is a valid id? I guess query form params get escaped anyway

Reply:
Yes, ids can contain any printable ASCII character so this seemed like the most straightforward approach.